google-cloud-dialogflow/samples/detect_intent_audio.rb

```diff
@@ -0,0 +1,81 @@
+# Copyright 2018 Google, Inc
```

Comment:
nit: probably should update to 2020 Google LLC

Reply:
fixed, here and in all other files.